client-super-turtle-tracker/src/components/reusables/TurtleListItem.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import { useEffect, useState } from "react";
+import { StyleSheet } from "react-native";
+import { Avatar, List } from "react-native-paper";
+import { View } from ".";
+import { firebase } from "../../../env";
+import { capitalizeFirstLetter } from "../../utils/helpers/functions";
+import { Turtle } from "../../utils/interfaces/Turtle";
+
+const styles = StyleSheet.create({
+  item: {
+    borderBottomColor: "#cdc",
+    borderBottomWidth: 0.5,
+  },
+});
+
+export const TurtleListItem = ({ turtle }: { turtle: Turtle }): JSX.Element => {
+  // TODO: Store url in database if possible
+  const getPhoto = async (photoName: string): Promise<string> => {
+    const ref = firebase.storage().ref().child(`images/${photoName}`);
+    const url = await ref.getDownloadURL();
+    return url;
+  };
```

Comment:
`getPhoto()` can be removed from the scope of the `TurtleListItem` component. In this instance, `getPhoto()` is redeclared on each render of `TurtleListItem`. This can be moved just above the function in this file, or added to the helper function file.